core/controllers/reader.py

```diff
@@ -183,9 +184,9 @@ def _get_exploration_player_data(
             interaction_templates),
         'is_private': rights_manager.is_exploration_private(
             exploration_id),
-        # Note that this overwrites the value in base.py.
+        # Note that this overwrites the value in base_classifier.py.
```

Comment:
Oh, that's pycharm refactoring at work. I had renamed base.py -> base_classifier.py and pycharm blindly replaced all the references in uncommitted files (not its fault). Thanks for catching that!